### DISCUSSION TITLE
[Feature] Share feature with import/dedup functionality

### DISCUSSION THREAD

**Discussion by blublub** [immich-app/immich#10360] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi

Currently when we share images with other immich users we can create an album on user A and share that with user B.

When user A however decides to delete images which are part of that album this change propagates to the album shared with B - so B is missing images he/she maybe wouldn't want to delete.

Would it be possible to import shared images in to person B's library without actually copying the image?  - so basically a "dedup" and if users A deletes that image it only disappears from users A library but it stays available for user B?

That would make sharing and keeping / deleting within ones own library very easy and save quite some disks space.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#1587
